elastic/apm-server
# [java-attacher] add --download-agent-version flag to invocation

## Motivation/summary

If DownloadAgentVersion is supplied in the config,
apm-server will pass it along to the java attacher
to download that agent version.

## Checklist





## How to test these changes

Start apm-server with `log.level=debug` and this option configured. Check the log output for the arguments used for starting the attacher.

## Related issues

closes #5853
